manual/php.md

```diff
@@ -401,6 +401,31 @@ class JFooHelper
 }
 ```
 
+### Namespaces
+
+Namespaces are formatted according to this flow. First there is the file docblock followed by the namespace the file lives in. After that we get the `defined` check and after that the
```

Comment:
One nit-pick.  The `defined or die` check is only a requirement inside the CMS environment, it's not required in the Framework repos or our apps built using the Framework.  So there should be some kind of comment along the lines of "when required, the `defined` check...".